config/config.go

```diff
@@ -679,6 +680,7 @@ type EC2SDConfig struct {
 	SecretKey       string   `yaml:"secret_key,omitempty"`
 	RefreshInterval Duration `yaml:"refresh_interval,omitempty"`
 	Port            int      `yaml:"port"`
+	SubnetSeparator string   `yaml:"subnet_separator,omitempty"`
```

Comment:
I am checking the Consul default tagSeparator [here](https://github.com/prometheus/prometheus/blob/master/config/config.go#L111) and it is a comma, however the default relabeling separator is a semicolon [here](https://github.com/prometheus/prometheus/blob/master/config/config.go#L95). Which one should I pick?